sdk/src/main/java/io/dapr/client/AbstractDaprClient.java

```diff
@@ -297,80 +298,80 @@
    */
   @Override
   public <T> Mono<State<T>> getState(
-      String stateStoreName, String key, String etag, StateOptions options, Class<T> clazz) {
-    return this.getState(stateStoreName, key, etag, options, TypeRef.get(clazz));
+          String storeName, String key, String etag, StateOptions options, Class<T> clazz) {
+    return this.getState(storeName, key, etag, options, TypeRef.get(clazz));
   }
 
   /**
    * {@inheritDoc}
    */
   @Override
-  public <T> Mono<List<State<T>>> getStates(String stateStoreName, List<String> keys, TypeRef<T> type) {
-    return this.getStates(new GetStatesRequestBuilder(stateStoreName, keys).build(), type).map(r -> r.getObject());
+  public <T> Mono<List<State<T>>> getBulkState(String storeName, List<String> keys, TypeRef<T> type) {
+    return this.getBulkState(new GetStatesRequestBuilder(storeName, keys).build(), type).map(r -> r.getObject());
   }
 
   /**
    * {@inheritDoc}
    */
   @Override
-  public <T> Mono<List<State<T>>> getStates(String stateStoreName, List<String> keys, Class<T> clazz) {
-    return this.getStates(stateStoreName, keys, TypeRef.get(clazz));
+  public <T> Mono<List<State<T>>> getBulkState(String storeName, List<String> keys, Class<T> clazz) {
+    return this.getBulkState(storeName, keys, TypeRef.get(clazz));
   }
 
   /**
    * {@inheritDoc}
    */
   @Override
-  public Mono<Void> executeTransaction(String stateStoreName,
-                                       List<TransactionalStateOperation<?>> operations) {
-    ExecuteStateTransactionRequest request = new ExecuteStateTransactionRequestBuilder(stateStoreName)
+  public Mono<Void> executeStateTransaction(String storeName,
+                                            List<TransactionalStateOperation<?>> operations) {
+    ExecuteStateTransactionRequest request = new ExecuteStateTransactionRequestBuilder(storeName)
         .withTransactionalStates(operations)
         .build();
-    return executeTransaction(request).then();
+    return executeStateTransaction(request).then();
   }
 
   /**
    * {@inheritDoc}
    */
   @Override
-  public Mono<Void> saveStates(String stateStoreName, List<State<?>> states) {
-    SaveStateRequest request = new SaveStateRequestBuilder(stateStoreName)
+  public Mono<Void> saveBulkStates(String storeName, List<State<?>> states) {
+    SaveStateRequest request = new SaveStateRequestBuilder(storeName)
         .withStates(states)
         .build();
-    return this.saveStates(request).then();
+    return this.saveBulkStates(request).then();
   }
 
   /**
    * {@inheritDoc}
    */
   @Override
-  public Mono<Void> saveState(String stateStoreName, String key, Object value) {
-    return this.saveState(stateStoreName, key, null, value, null);
+  public Mono<Void> saveBulkState(String storeName, String key, Object value) {
```

Comment:
do not rename this. this will remain `saveState` similar to the `getState` counterparts.

Reply:
Correct. This is singular.